internal/config/config_js.go

```diff
@@ -0,0 +1,65 @@
+//go:build js || wasm
+
+package config
+
+import (
+	"gonum.org/v1/gonum/spatial/r2"
+	"strconv"
+	"syscall/js"
+)
+
+type Config struct {
+	DefaultWindowSize r2.Vec
+	Ratio             float64
+	TargetTPS         int
+	Debug             bool
+}
+
+var (
+	C *Config
+)
+
+func init() {
+	defaultConfiguration := getDefaultConfiguration()
+	mergedConfiguration := getLocalStorageConfiguration(defaultConfiguration)
+
+	C = mergedConfiguration
+}
+
+func getDefaultConfiguration() *Config {
+	return &Config{
```

Comment:
Our storage model is actually different from our application model, because `Ratio` is a calculated value. I think it would be nice to make that clear in the code.